src/main/webapp/views/COMMONS/searchBarVendors.html

```diff
@@ -0,0 +1,32 @@
+<div class="row justify-content-md-center">
+  <div class="col-12">
+    <div class="card border-primary">
+      <h5 class="card-header">Find the nearest vendor</h5>
+      <div class="card-body">
+        <div id="search-form">
+          <div class="row">
+            <div class="col">
+              <div class="form-group">
+                <label for="distance">Distance in meters:</label>
+                <input type="number" class="form-control" id="distance" aria-describedby="distance"
+                  placeholder="Distance" value="1000">
```

Comment:
I think you can provide a min and max value, right?
What could be a good max distance? 🤔

Reply:
I think the distance should not exceed the 3km (3000 m), even that seems a little too much to me considering is the radius of the costumers search area :thinking:

Reply:
Sure we can set that